students/alexLaws/lesson05/mailroom.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+
+import os
+
+donor_dictionary = {
+    'Ryan Moore': [500, 250], 'Ted Laws': [1000, 100], 'Ben Snell': [150],
+    'Andrew Crawford': [2000, 2000, 4000], 'Beth Ross': [400]}
+
+
+prompt = ("\nWhat would you like to do?\n"
+          "Choose an action from this list:\n"
+          "1 - Add a New Donation to the Records\n"
+          "2 - Send a Thank You\n"
+          "3 - Thank Everyone\n"
+          "4 - Create a Report\n"
+          "5 - Quit\n")
+
+
+def menu_selection(prompt, dispatch_dict):
+    while True:
+        response = input(prompt)
+        try:
+            if dispatch_dict[response]() == "Exit Menu":
+                break
+        except KeyError:
+            print("\nThat was not one of the options.")
+
+
+def exit():
+    return "Exit Menu"
+
+
+def report():
+    print("\nDonor Name       |  Total Given  |  Num Gifts  |  Average Gift")
+    donor_info = [{'name': i,
+                   'tot': sum(donor_dictionary[i]),
+                   'num': len(donor_dictionary[i]),
+                   'avg':sum(donor_dictionary[i]) / len(donor_dictionary[i])}
+                  for i in donor_dictionary]
+    from operator import itemgetter
```

Comment:
It's standard in Python to place all imports at the top of the file so you can quickly scan what packages are included